author.rb

```diff
@@ -0,0 +1,20 @@
+require_relative './item'
+# author class
+
+class Author
+  attr_reader :id, :frist_name, :last_name, :items
+
+  def initialize(frist_name, last_name)
+    @id = Random.rand(1..1000)
+    @frist_name = frist_name
```

Comment:
There is a typo on `@frist_name = frist_name`
Typos can be a source of bugs in a codebase.
We could change it to `first_name`

Reply:
Hi, @jssol. I will change it before merging. Thanks for you note 😄